Adding request id($rid) formate (#102263)
Azure/azure-content: articles/iot-hub/iot-hub-devguide-direct-methods.md

* The `status` property is the device-supplied status of method execution.

> * The `$rid` property is the request ID from the method invocation received from IoT Hub and request ID contain value in hexadecimal formate.

Reviewer: @SumitBalia Please make this editorial fix, and then I can approve this pull request. Thank you.

```suggestion
* The `$rid` property is the request ID from the method invocation received from IoT Hub. The request ID is a hexadecimal formatted value.
```

Author: Thank you @kgremban for your suggestion. Done the changes based on your suggestion.
